### PR TITLE
Session manager

### PR DESCRIPTION
**Pull request by Zer0SZN**
## What type of PR is this? (check all applicable)

- [ ] Feature
- [ ] Refactor
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update
- [x] Other: Session Management Function
  
  
## Description of what this Pull Request does
- **`SessionManagement.js`** function manages a session for two users, tracking their join time and ensuring session capacity is not exceeded.
## Related Tickets & Documents
- Closes https://github.com/annonymousauthority/P2PCoder/issues/130

### CODE REVIEW

File: src/composables/sessionManagement.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+function storeSessionData(users) {
+  localStorage.setItem("users", JSON.stringify(users));
+}
+
+function getSessionData() {
+  const users = JSON.parse(localStorage.getItem("users"));
+  return users;
+}
+
+function startTimer(joinTime) {
```
Comment (annonymousauthority) [inline src/composables/sessionManagement.js:13]: Why is there a join time, if a new time is initialized in the function

**Review by annonymousauthority — CHANGES_REQUESTED** [review]
Let's have a look at this function tmr

**Comment by Zer0SZN** [comment]
> Let's have a look at this function tmr

Ok

File: src/composables/sessionManagement.js (hunk 1 above)
Comment (Zer0SZN) [reply src/composables/sessionManagement.js:13]: The join time is when the a user joins the session

**Comment by Zer0SZN** [comment]
The join time is when the a user joins the session 


The startTimer function calculates the time difference between the current time and the recorded join time, terminating the session after 59 minutes and 59 seconds.
 
The current time which is obtained using new Date() to get the current date and time, which is used to track the duration of the session and determine when the session should be terminated based on a predefined time limit

**Pull request closed without merging**